modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/process-metrics/performance-by-step-card/PerformanceByStepCard.es.js

```diff
@@ -22,7 +22,7 @@ import TimeRangeFilter from '../../filter/TimeRangeFilter.es';
 import {getTimeRangeParams} from '../../filter/util/timeRangeUtil.es';
 import {Body, Footer} from './PerformanceByStepCardBody.es';
 
-function Header({disableFilters, prefixKey, processId, totalCount}) {
+function Header({disableFilters, prefixKey, processId}) {
```

Comment:
Hey @naraandrade, Did you check why someone put the `totalCount` here? It seems that was a workaround fix but I didn't remember why. Thx

Reply:
@rafaprax it was only used to disable the filter in case there were no items to be displayed. But we cannot disable the filter based on that because, otherwise, the user wouldn't be able to choose another filter after that.

Reply:
ok then. Thx for the clarification